package.json

```diff
@@ -7,8 +7,10 @@
   "license": "Apache-2.0",
   "repository": "zooniverse/panoptes-javascript-client",
   "dependencies": {
-    "json-api-client": "^7.0.2",
-    "local-storage": "^2.0.0"
+    "normalizeurl": "~1.0.0",
+    "local-storage": "^2.0.0",
+    "sugar-client": "^2.0.0",
```

Comment:
SugarClient is part of of PJC now?

Reply:
Good point. It wasn’t when I opened this PR, and this branch needs updating.